specification/pom.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ 
+    Copyright (c) 2019 Payara Services Limited. All rights reserved.
```

Comment:
Should this be switched to,
`Copyright (c) 2019 Contributors to the Eclipse Foundation` ?

Reply:
I switched it